### PR TITLE
1157-group-purpose

### PR DESCRIPTION
**Pull request by tibetsprague**
Fixes https://github.com/Hylozoic/hylo-evo/issues/1157

### CODE REVIEW

File: src/components/CreateGroup/CreateGroup.js [hunk 1]
```diff
@@ -250,6 +256,22 @@ class CreateGroup extends Component {
...
 
+        <div styleName='purposeContainer'>
+          <div styleName='purposeField'>
+            <span styleName='title'>{t('GROUP PURPOSE')}</span>
+            <span styleName='characterCounter'>{purposeCharacterCount} / 500</span>
+            <div styleName='purposeHelp'>
+              ?
+              <div styleName='purposeTooltip'>{t('purposeHelpText')}</div>
+            </div>
+            <textarea
```
Comment (thomasgwatson) [inline src/components/CreateGroup/CreateGroup.js:267]: So no fancy styling via HyloEditor?

File: src/routes/GroupDetail/GroupDetail.js [hunk 2]
```diff
@@ -213,6 +213,11 @@ class UnwrappedGroupDetail extends Component {
             </div>
           ) : (
             <div styleName='g.groupDescription'>
+              <h3>{t('Purpose')}</h3>
+              <ClickCatcher>
+                <HyloHTML element='span' html={TextHelpers.markdown(group.purpose)} />
```
Comment (thomasgwatson) [inline src/routes/GroupDetail/GroupDetail.js:218]: Ahh so we can display fancy text...

**Review by thomasgwatson — APPROVED** [review]
Lgtm, just added some comments about whether we are doing rich-text/fancy text or not

File: src/routes/GroupDetail/GroupDetail.js (hunk 2 above)
Comment (tibetsprague) [reply src/routes/GroupDetail/GroupDetail.js:218]: Yeah I think we decided not to promote fancy text but allow it via markdown

**Pull request merged**